src/hooks/useKeyboardControls.ts

```diff
@@ -34,6 +35,15 @@ export const useKeyboardControls: ReactHook<UseKeyboardControlsProps, void> = ({
   const { profile } = useUser();
   const { row, column } = profile?.data?.[venueId] ?? {};
 
+  const { enterRoom } = useRoom(room);
```

Comment:
Where is `room` coming from for this?

Not sure if we would necessarily want this hook within this function, do we already use it in the component that calls `useKeyboardControls` or similar? Could we maybe just pass down `enterRoom` through the props or similar?

Reply:
changed in https://github.com/sparkletown/sparkle/pull/1129/commits/53f57c40e24983470b7552d4ebed6f1715ff74ba